worlds/sc2/items.py

```diff
@@ -907,6 +907,9 @@ def get_full_item_list():
     item_names.PSI_DISRUPTER:
         ItemData(619 + SC2WOL_ITEM_ID_OFFSET, TerranItemType.Building, 6, SC2Race.TERRAN,
                  classification=ItemClassification.progression),
+    item_names.DEVASTATOR_TURRET:
+        ItemData(626 + SC2WOL_ITEM_ID_OFFSET, TerranItemType.Building, 7, SC2Race.TERRAN,
+                 classification=ItemClassification.progression),
```

Comment:
Use id 403 (where most buildings are) and put it into correct place

Psi Disrupter/etc were previously in Laboratory (due to vanilla unlock) but moved to building group later. The old IDs can't be changed due to backward compatibility (otherwise I'd relocate a lot of stuff)